variables-common.tf

```diff
@@ -16,6 +16,12 @@ locals {
   )
 }
 
+variable "timezone" {
```

Comment:
Please prefix with `default_worker`

Reply:
This is used for workers, bastion, and operator instances - updated the description.